CustomStatusChangedTrigger.cls

```diff
@@ -0,0 +1,6 @@
+trigger CustomStatusChangedTrigger on Class__c (after update) {
+  
+    CustomStatusChanged.ValueChangedToReset(trigger.new,trigger.oldmap);

```

Comment:
why two argument ?

Reply:
First one for accessing the updating list and second one for getting old results of that object.